src/Utils.ps1

```diff
@@ -176,6 +176,14 @@ function Add-PoshGitToProfile {
         $profileContent = "`nImport-Module '$ModuleBasePath\posh-git.psd1'"
     }
 
+    # Make sure the profile path exists
+    $profileDir = Split-Path -Parent $profilePath
```

Comment:
Sorry missed one other minor issue (which I myself forget from time-to-time).  Can you switch the order on params here and use `-LiteralPath` e.g.` $profileDir = Split-Path -LiteralPath $profilePath -Parent`.  Thanks.